lib/providers/twitter.js

```diff
@@ -4,6 +4,8 @@ exports = module.exports = function (options) {
 
     options = options || {};
```

Comment:
Let's clean this up a bit now that it's getting slightly more complex
```
const defaults = {
    extendedProfile: true,
    getMethod: 'users/show'
};
const settings = Hoek.applyToDefaults(defaults, options);
```

Don't forget to require hoek and change options to settings.